elrond_dex_farm/src/lib.rs

```diff
@@ -9,6 +9,7 @@ type Nonce = u64;
 const PENALTY_PERCENT: u64 = 10;
 const BURN_TOKENS_GAS_LIMIT: u64 = 5000000;
 const EXIT_FARM_NO_PENALTY_MIN_EPOCHS: u64 = 3;
+const LOCKED_REWARDS_LIQUIDITY_MUTIPLIER: u64 = 2;
```

Comment:
we need functions through which these values can be set. Also this value has to be written in the created NFT for the farming position.

Reply:
done